client/assets/js/main.js

```diff
@@ -2,42 +2,50 @@
 
 import Board from './board';
 import server from './sendToServer.js';
+import $ from 'jquery';
 
 document.addEventListener('DOMContentLoaded', function () {
 
-	let board = new Board();
+	$('.username').submit(function(e) {
+		e.preventDefault();
+		var name = $(this).find('.field-username')[0].value;
 
+		$(this).hide();
```

Comment:
Sorry, I should have caught this earlier, but should this be `.remove`? Is it ever shown again?